helm-charts/codegen/values.yaml

```diff
@@ -105,6 +105,13 @@ codegen-ui:
 #   service:
 #     type: ClusterIP
 
+# External LLM configuration
+externalLLM:
+  enabled: false
+  LLM_SERVICE_HOST_IP: "http://your-llm-server"
+  LLM_MODEL: "your-model"
+  OPENAI_API_KEY: "your-api-key"
+
```

Comment:
Please add default values for all the newly added conditional dependency in Chart.yaml

Reply:
@lianhao are the default values for `tei` and `redis-vector-db` satisfied by `retriever-usvc` and `embedding-usvc`

Reply:
nope. We should explicitly set default values in `values.yaml` for any conditional dependency added in Chart.yaml